### PR TITLE
fixed PROJQUAY-730

### PR DESCRIPTION
**Pull request by jasinner**

### CODE REVIEW

File: modules/proc_deploy_quay_openshift.adoc [hunk 1]
```diff
@@ -137,12 +137,6 @@ $ oc create -f quay-servicetoken-role-k8s1-6.yaml
 $ oc create -f quay-servicetoken-role-binding-k8s1-6.yaml
 ```
 
-. **Add privilege**: Make sure that the service account has root privileges, because {productname} runs strictly under root (this will be changed in the future versions). Throughout this example, the namespace is `quay-enterprise`:
-+
-```
-$ oc adm policy add-scc-to-user anyuid \
-     system:serviceaccount:quay-enterprise:default
-```
```
Comment (thomasmckay) [inline modules/proc_deploy_quay_openshift.adoc:145]: The `quay` container runs as non-root now but `clair-jwt` still runs as root. Without the policy above, does `clair-jwt` still work?

Comment (jasinner) [reply modules/proc_deploy_quay_openshift.adoc:145]: I haven't tried installing Clair yet. I'll give it a go in the coming days. Do we have an existing Jira ticket to get clair-jwt running as non-root?

Comment (jasinner) [reply modules/proc_deploy_quay_openshift.adoc:145]: Found the [existing issue|https://issues.redhat.com/browse/QUAY-2087] for `clair-jwt`.

Comment (jasinner) [reply modules/proc_deploy_quay_openshift.adoc:145]: Added another commit which adds a clair-jwt SA for that privilege in clair.

**Pull request merged**